sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumer/EventHubConsumerTests.cs

```diff
@@ -1188,7 +1188,7 @@ await foreach (EventData eventData in consumer.SubscribeToEvents(cancellation.To
         private int GetBackgroundPublishReceiveBatchSize(EventHubConsumer consumer) =>
             (int)
                 typeof(EventHubConsumer)
-                    .GetField("_backgroundPublishReceiveBatchSize", BindingFlags.Static | BindingFlags.NonPublic)
+                    .GetField("BackgroundPublishReceiveBatchSize", BindingFlags.Static | BindingFlags.NonPublic)
```

Comment:
Generally not a fan of probing internals for variable values, but I don't know its use-case in this test.

Reply:
I agree in general.  We try to limit the test surface to only the public interface, but there are a few cases where we do not want to expose things as `protected` to avoid leaking implementation details as external-visible.   To get around that, we're accepting the bit of brittleness that comes from using the private member with the test context.